extensions/jython/src/com/google/refine/jython/JythonEvaluable.java

```diff
@@ -103,17 +106,24 @@ public Evaluable parse(String s) throws ParsingException {
         logger.debug("Done with static block in Jython initialization");
     }
 
-    public JythonEvaluable(String s) {
+    // Convenience constructor for tests
+    protected JythonEvaluable(String source) {
```

Comment:
That's a change of public interface, but in an extension, so from my perspective this is fine.